dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -216,4 +223,13 @@ http.compression.types=List of content types for response compression
 http.compression.types.desc=To include a content type in addition to the default values, affix the add (+) character as a prefix to that content type. To exclude a content type from compression, affix the remove (-) character as a prefix to that content type. Note: The wildcard (*) character is supported only as a content subtype, such as text/*.
 
 http.compression.serverPreferredAlgorithm=Server-preferred algorithm for compression
-http.compression.serverPreferredAlgorithm.desc=The configured compression algorithm is used to compress the body of responses when it is specified with a non-zero quality value in the request's Accept-Encoding header. The valid compression algorithms include: deflate, gzip, x-gzip, zlib, and identity.
\ No newline at end of file
+http.compression.serverPreferredAlgorithm.desc=The configured compression algorithm is used to compress the body of responses when it is specified with a non-zero quality value in the request's Accept-Encoding header. The valid compression algorithms include: deflate, gzip, x-gzip, zlib, and identity.
+
+http.cookies.samesite.lax=Lax cookies
+http.cookies.samesite.lax.desc=List of cookies which should have the value of Lax set to the SameSite attribute if not already defined.
+
+http.cookies.samesite.none=None cookies
+http.cookies.samesite.none.desc=List of cookies which should have the value of None set to the SameSite attribute if not already defined.
```

Comment:
same desc edits as above:
List of cookies that should have the SameSite attribute set to a value of "None", if not already defined.